lib/src/neu/generators/openapi3/openapi3-schema.ts

```diff
@@ -0,0 +1,500 @@
+// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#openapi-object
+export interface OpenApiV3 {
```

Comment:
Out of scope of this PR - but should we move these type definitions out of the Spot codebase and create a separate package? My main concern here is that the codegen leverages these types and duplicates them. I feel like it's getting hard to maintain.

Reply:
As long as the definition used in any project is compatible with OpenAPI it shouldn't matter. However, it'd be a good idea to use exactly the same definitions. Perhaps after the dust settles on the generator migrations.